src/etc/inc/auth.inc

```diff
@@ -2171,4 +2171,102 @@ function get_user_remote_authsource() {
 	}
 	return $authsource;
 }
+function set_pam_auth() {
+	global $config;
+
+	$pam_sshd =<<<EOD
+#
+# \$FreeBSD$
+#
+# PAM configuration for the "sshd" service
+#
```

Comment:
Do we need to include all of the comments and disabled/commented-out lines? This should probably instead have a header saying it's automatically generated and that it shouldn't be edited. Similar to other config files we generate. (Same goes for the other files in this PR)

Reply:
> his should probably instead have a header saying it's automatically generated and that it shouldn't be edited.
Added

> If that is the case, the GUI needs to warn about that.
Added

> There is a `template_user` option to `pam_radius` which may help but that seems like a different problem source. If that path is taken then we'd need to find/detect/something the automatically generated `passwd` entries and show them in the GUI.
Better to use pam LDAP for that

> Also need to make sure this works with `sudo`.
`sudo` pam already includes system template,-
/usr/local/etc/pam.d/sudo:
```
#
# $FreeBSD$
#
# PAM configuration for the "sudo" service
#

# auth
auth		include		system

# account
account		include		system

# session
# XXX: pam_lastlog (used in system) causes users to appear as though
# they are no longer logged in in system logs.
session		required	pam_permit.so

# password
password	include		system
```

Reply:
+ it seems safe to remove
```
auth		sufficient	pam_opie.so		no_warn no_fake_prompts
auth		requisite	pam_opieaccess.so	no_warn allow_local
```